### ISSUE TITLE
'bootstrap.link' File Not Found

### ISSUE THREAD

**Issue by Zhuohao-Li** [LemonOSProject/LemonOS#35]
Hi, I have a problem after Docker Set up following Build Lemon OS document. When I want to build and run `xbstrap install --all ` in docker container, error happens like this `FileNotFoundError: [Errno 2] No such file or directory: 'bootstrap.link'`. I'm new to Operating System, so pretty much thanks for checking my issue

**Comment by fido2020** [comment]
Have you followed all steps? Running `xbstrap init ..` from `Build/` will create the bootstrap.link file

**Comment by Zhuohao-Li** [comment]
Yep, I initialized docker container for it. But I can’t find Build/ in my root category. I don’t know why, even sudo is not permitted.
***@***.***
发件人: fido2020 ***@***.***>
日期: 星期三, 2022年3月16日 09:13
收件人: LemonOSProject/LemonOS ***@***.***>
抄送: Edith ***@***.***>, Author ***@***.***>
主题: Re: [LemonOSProject/LemonOS] 'bootstrap.link' File Not Found (Issue #35)

Have you followed all steps? Running xbstrap init .. from Build/ will create the bootstrap.link file

―

**Comment by fido2020** [comment]
There was an issue where xbstrap didn't find the Docker container, it should be fixed so pull and re-run `docker-init.sh` and let me know if it works.

**Comment by Zhuohao-Li** [comment]
I just re-run docker-init.sh in Scripts/, bu when I get into the environment and run ‘whereis Build’, it shows there’s no corresponding folder.
***@***.***
发件人: fido2020 ***@***.***>
日期: 星期三, 2022年3月16日 17:18
收件人: LemonOSProject/LemonOS ***@***.***>
抄送: Edith ***@***.***>, Author ***@***.***>
主题: Re: [LemonOSProject/LemonOS] 'bootstrap.link' File Not Found (Issue #35)

There was an issue where xbstrap didn't find the Docker container, it should be fixed so re-run docker-init.sh and let me know if it works.

―
